master/buildbot/data/changes.py

```diff
@@ -42,6 +42,7 @@ def _fixChange(self, change, is_graphql):
                     {'name': k, 'source': v[1], 'value': json.dumps(v[0])}
                     for k, v in props.items()
                 ]
+                change['builds'] = yield self.master.db.builds.getBuildsForChange(change['changeid'])
```

Comment:
Just out of interest, are you using the graphql API?

Reply:
no, not using graphql